lib/hawkular/base_client.rb

```diff
@@ -174,14 +174,16 @@ def tenant_header
     end
 
     def handle_fault(f)
+      http_code = (f.respond_to?(:http_code) ? f.http_code : 0)
+      fail HawkularException.new('Invalid credentials', http_code) if f.instance_of? RestClient::Unauthorized
```

Comment:
It sounds to me that in error cases the api should not return HTML from the server. @pilhuhn wdyt?